view/rooms/index.pug

```diff
@@ -0,0 +1,35 @@
+//
+  Copyright (c) 2017, salesforce.com, inc.
+  All rights reserved.
+  Licensed under the BSD 3-Clause license.
+  For full license text, see LICENSE.txt file in the repo root or
+  https://opensource.org/licenses/BSD-3-Clause
+
+doctype html
+html
+  head
+    title Refocus Room
+    link(rel='shortcut icon', href='favicon.ico')
+    link(rel='stylesheet', type='text/css', href='../static/css/salesforce-lightning-design-system.min.css')
+    script(src='http://code.jquery.com/jquery-1.11.1.js')
```

Comment:
load only https resources - this will fail the page to load when hosted on SSL enabled servers

Reply:
I thought that the SSL only restriction would only apply to incoming requests.
Also, if jquery is required, maybe better to do the require in the code and let webpack bundle it up?

Reply:
I can't prove it easily (lack of ssl-enabled dev env), but I'm 99% it would fail to load.

See: "An HTTPS page that includes content fetched using cleartext HTTP is called a mixed content page" [..] "Pages like this are only partially encrypted" [..] "That leaves the pages unsafe." [...] "Starting in Firefox 23, mixed active content is blocked by default" 
from https://developer.mozilla.org/en-US/docs/Web/Security/Mixed_content